README.md

```diff
@@ -27,25 +27,25 @@ complex, and error-prone.
 
 ## Table of contents
 
-* [Getting started](#getting-started)
-   * [RSpec](#rspec)
-   * [Minitest](#minitest)
-* [Usage](#usage)
-  * [On the subject of `subject`](#on-the-subject-of-subject)
-  * [Availability of RSpec matchers in example groups](#availability-of-rspec-matchers-in-example-groups)
-  * [`should` vs `is_expected.to`](#should-vs-is_expectedto)
-* [Matchers](#matchers)
-   * [ActiveModel matchers](#activemodel-matchers)
-   * [ActiveRecord matchers](#activerecord-matchers)
-   * [ActionController matchers](#actioncontroller-matchers)
-   * [Independent matchers](#independent-matchers)
-* [Extensions](#extensions)
-* [Contributing](#contributing)
-* [Compatibility](#compatibility)
-* [Versioning](#versioning)
-* [Team](#team)
-* [Copyright/License](#copyright-license)
-* [About thoughtbot](#about-thoughtbot)
+- [Getting started](#getting-started)
```

Comment:
Are the formatting changes to the README necessary in this PR or can we do this in a different PR? Is there something that forced this change?

Reply:
Oh, my bad. It was the linter :sweat_smile: Lemme fix it